### ISSUE TITLE
Question about epoch of training nerf

### ISSUE THREAD

**Issue by shenyehui** [ActiveVisionLab/DFNet#20]
Dear author, hello. In your code, you have set the number of epochs for training nerf to 600. Is it necessary for the model to reach 600 epochs to achieve the best performance under normal circumstances?

**Comment by chenusc11** [comment]
Hi, I think it is enough to get a reasonable quality NeRF for 7-scene and Cambridge in our setting. However, if you use your own dataset with different amount of data, you might want to tune the hyper-parameters.

**Comment by shenyehui** [comment]
> Hi, I think it is enough to get a reasonable quality NeRF for 7-scene and Cambridge in our setting. However, if you use your own dataset with different amount of data, you might want to tune the hyper-parameters.

Thank you for your answer, but I noticed during training that after more than 200 epochs, the loss and PSNR don't seem to improve significantly, I would like to ask if there is a big difference between the models produced by 200, 400 and 600 epochs?

**Comment by chenusc11** [comment]
Hi, indeed, in my experience, 200 is already pretty reasonable. 600 will improve the nerf model a bit more (on PSNR) but not significantly.

**Comment by shenyehui** [comment]
> Hi, indeed, in my experience, 200 is already pretty reasonable. 600 will improve the nerf model a bit more (on PSNR) but not significantly.

Thank you!
